Src/AutoFixture.ruleset

```diff
@@ -1,236 +1,247 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<RuleSet Name="AutoFixture" Description="Rule set for all AutoFixture projects" ToolsVersion="10.0">
+<RuleSet Name="AutoFixture" Description="Rule set for all AutoFixture projects" ToolsVersion="15.0">
   <IncludeAll Action="Warning" />
   <Rules AnalyzerId="Microsoft.Analyzers.ManagedCodeAnalysis" RuleNamespace="Microsoft.Rules.Managed">
-    <Rule Id="CA1000" Action="Error" />

```

Comment:
I would expect non of the Rule Ids to change. What's the motivation behind these changes?